test/TestUtilities/Test.Utility/Signing/TestCertificateGenerator.cs

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.ObjectModel;
+using System.Security.Cryptography;
+using System.Security.Cryptography.X509Certificates;
+
+namespace Test.Utility.Signing
+{
+    public class TestCertificateGenerator
+    {
+        public static readonly Oid IdKPCodeSigning = new Oid("1.3.6.1.5.5.7.3.3");
```

Comment:
Test OID's should go in their own static class like we do [here](https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Core/NuGet.Packaging/Signing/Oids.cs).

Also, for code signing you can use `NuGet.Packaging.Signing.Oids.CodeSigningEku` instead of redefining it here.

Also, you're not obliged to keep BouncyCastle names.  I'd change `IdKP` to `IdKp`.